include/langinfo.h

```diff
@@ -0,0 +1,126 @@
+#define DAY_3               NL_ITEM(LC_TIME,     0x09)
+#define DAY_4               NL_ITEM(LC_TIME,     0x0a)
+#define DAY_5               NL_ITEM(LC_TIME,     0x0b)
+#define DAY_6               NL_ITEM(LC_TIME,     0x0c)
+#define DAY_7               NL_ITEM(LC_TIME,     0x0d)
+
+#define ABMON_1             NL_ITEM(LC_TIME,     0x0e)
+#define ABMON_2             NL_ITEM(LC_TIME,     0x0f)
+#define ABMON_3             NL_ITEM(LC_TIME,     0x10)
+#define ABMON_4             NL_ITEM(LC_TIME,     0x11)
+#define ABMON_5             NL_ITEM(LC_TIME,     0x12)
+#define ABMON_6             NL_ITEM(LC_TIME,     0x13)
+#define ABMON_7             NL_ITEM(LC_TIME,     0x14)
+#define ABMON_8             NL_ITEM(LC_TIME,     0x15)
+#define ABMON_9             NL_ITEM(LC_TIME,     0x16)
+#define ABMON_10            NL_ITEM(LC_TIME,     0x17)
+#define ABMON_11            NL_ITEM(LC_TIME,     0x18)
+#define ABMON_12            NL_ITEM(LC_TIME,     0x19)
+
+#define MON_1               NL_ITEM(LC_TIME,     0x1a)
+#define MON_2               NL_ITEM(LC_TIME,     0x1b)
+#define MON_3               NL_ITEM(LC_TIME,     0x1c)
+#define MON_4               NL_ITEM(LC_TIME,     0x1d)
+#define MON_5               NL_ITEM(LC_TIME,     0x1e)
+#define MON_6               NL_ITEM(LC_TIME,     0x1f)
+#define MON_7               NL_ITEM(LC_TIME,     0x20)
+#define MON_8               NL_ITEM(LC_TIME,     0x21)
+#define MON_9               NL_ITEM(LC_TIME,     0x22)
+#define MON_10              NL_ITEM(LC_TIME,     0x23)
+#define MON_11              NL_ITEM(LC_TIME,     0x24)
+#define MON_12              NL_ITEM(LC_TIME,     0x25)
+
+#define AM_STR              NL_ITEM(LC_TIME,     0x26)
+#define PM_STR              NL_ITEM(LC_TIME,     0x27)
+#define D_T_FMT             NL_ITEM(LC_TIME,     0x28)
+#define D_FMT               NL_ITEM(LC_TIME,     0x29)
+#define T_FMT               NL_ITEM(LC_TIME,     0x2a)
+#define T_FMT_AMPM          NL_ITEM(LC_TIME,     0x2b)
+#define ERA                 NL_ITEM(LC_TIME,     0x2c)
+#define ERA_D_FMT           NL_ITEM(LC_TIME,     0x2e)
+#define ALT_DIGITS          NL_ITEM(LC_TIME,     0x2f)
+#define ERA_D_T_FMT         NL_ITEM(LC_TIME,     0x30)
+#define ERA_T_FMT           NL_ITEM(LC_TIME,     0x31)
+
+#define CRNCYSTR            NL_ITEM(LC_MONETARY, 0x0f)
+
+#define YESEXPR             NL_ITEM(LC_MESSAGES, 0x00)
+#define NOEXPR              NL_ITEM(LC_MESSAGES, 0x01)
+#define YESSTR              NL_ITEM(LC_MESSAGES, 0x02)
+#define NOSTR               NL_ITEM(LC_MESSAGES, 0x03)
+
+#define nl_langinfo_l(i, l) nl_langinfo(i)
+
+/****************************************************************************
+ * Public Function Prototypes
+ ****************************************************************************/
+
+#ifdef __cplusplus
+#define EXTERN extern "C"
+extern "C"
+{
+#else
+#define EXTERN extern
+#endif
+
+FAR char *nl_langinfo(nl_item item);
+FAR char *nl_langinfo_l(nl_item item, locale_t l);
```

Comment:
`nl_langinfo_l` Is already implemented as a macro, this prototype declaration should not be required.

Reply:
Done.